src/configreader.cpp

```diff
@@ -12,20 +10,20 @@ void asteConfig::load(const std::string &asteConfigFile)
   try {
     preciceConfigFilename = config["precice-config"].get<std::string>();
   } catch (nlohmann::detail::parse_error) {
-    std::cerr << "Error while parsing ASTE configuration file \"precice-config\" is missing\n";
+    ASTE_ERROR << "Error while parsing ASTE configuration file \"precice-config\" is missing";
     MPI_Abort(MPI_COMM_WORLD, EXIT_FAILURE);
   } catch (nlohmann::detail::type_error) {
-    std::cerr << "Error while parsing ASTE configuration file \"precice-config\" is missing\n";
+    ASTE_ERROR << "Error while parsing ASTE configuration file \"precice-config\" is missing";
```

Comment:
Are you sure that the newline statement should be removed here?

Reply:
Boost log automatically adds newlines there is no need to newline statements at the end.